### PR TITLE
USHIFT-644: Skip sig-cloud-provider tests

### PR DESCRIPTION
**Pull request by pacevedom**

### CODE REVIEW

File: test/extended/cloud_controller_manager/ccm.go [hunk 1]
```diff
@@ -20,7 +20,7 @@ var _ = g.Describe("[sig-cloud-provider][Feature:OpenShiftCloudControllerManager
 	defer g.GinkgoRecover()
 	oc := exutil.NewCLI("ccm")
 
-	g.It("Deploy an external cloud provider", func() {
+	g.It("Deploy an external cloud provider [apigroup:config.openshift.io][apigroup:machineconfiguration.openshift.io]", func() {
```
Comment (ingvagabund) [inline test/extended/cloud_controller_manager/ccm.go:23]: I assume `machineconfiguration.openshift.io` comes from running `oc get machineconfig`.

Comment (pacevedom) [reply test/extended/cloud_controller_manager/ccm.go:23]: Exactly.

**Pull request merged**